compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -233,6 +233,29 @@ function parseFunctionTypeDesc(Tokenizer tok, FunctionParam[]? namedParams = ())
                 params.push(param);
                 check tok.advance();
             }
+            "..." => {
+                check tok.advance();
+                Token? t = tok.current();
+                if t is [IDENTIFIER, string] {
```

Comment:
Too much repeated code here. This needs refactoring.

Reply:
fixed